### DISCUSSION TITLE
Can I get the wine terminals to run quietly?

### DISCUSSION THREAD

**Discussion by ghost** [DeckCheatz/wemod-launcher#107] [Q&A] [answered]
Was wondering if there was a way to make the wine terminals run in the background so that the only windows that pop up are wemod.

**Comment by marvin1099** [comment, marked as answer]
The terminal is there for debug and to see if things work,
It might be possible to run this in the backround, 
but since this will only appear at game start and will then be behind the game its not that needed to be hidden.
If the console stays in front for you say so that would be a problem then.

**Comment by marvin1099** [comment]
I assume this has awnsered the question so I will close this discussion.

For some additional info,
if you are on steamos enable devoper mode then you can switch windows using the steam button,
On a normal Linux desktop you can switch windows by using alt+tab, check your distro info for that if it won't work
